server/src/models/services/populate.js

```diff
@@ -148,14 +148,14 @@ export default async function ({ models }) {
   );
 
   const raw_service_rows = get_standard_csv_file_rows("services.csv");
-  const most_recent_submission_year = _.chain(raw_service_rows)
+  const absolute_most_recent_submission_year = _.chain(raw_service_rows)
     .map("dept_submissions__document__year")
     .uniq()
     .sort()
     .last()
     .value();
 
-  const service_rows = _.chain(raw_service_rows)
+  const filtered_service_rows = _.chain(raw_service_rows)
```

Comment:
These aren't filtered

Reply:
Isn't this filtering for the last submission year?
```
    .map((service_across_years) =>
      _.chain(service_across_years)
        .sortBy(({ submission_year }) => _.toInteger(submission_year))
        .last()
        .value()
    )
```

Reply:
line 275

Reply:
Ohh, yeah, my bad. Yeah, that rejects all duplicates from old years. I still don't think of this as "filtered", since its what ends up as the canonical rows in the DB. The filtering is just to reject data we aren't currently set up to do anything with.

Reply:
Still, "rows" in `service_rows`, is different from the "rows" represented in raw data (`services.csv`) because it "filters" out service rows that are not the most recent submission year. IMO just `service_rows` doesn't clearly show that.
Is this rejecting previous years temporary so that you just want this to be called `service_rows`?

Reply:
The DB will only ever have one document ("row") per service. The end state will be by-year sub-documents off that to hold the year data. 

It's a matter of perspective here, and I guess whether you were referring to csv rows or DB "rows" in the name. It's filtered from the perspective of the csv rows, but the content of the variable represents eventual database "rows" (from that perspective it's not filtered, it is the full set of rows the database will know about).

Talking about rows in mongo is inaccurate (and just confusing here anyway, haha). I think I usually refer to the data as "records" once it's been transformed from the raw csv rows, but I'm sure I've not always been consistent on that.